core/modules/layout/layout.admin.inc

```diff
@@ -1102,21 +1102,69 @@ function layout_settings_form_save_submit($form, &$form_state) {
     unset($_SESSION['layout_new_name']);
   }
 
-  // If this is a new layout, populate blocks from default layout.
+  // If this is a new layout, populate it with blocks from the default layout.
   if (!empty($layout->is_new)) {
     $default_layout_name = path_is_admin($layout->getPath()) ? 'admin_default' : 'default';
     $default_layout = layout_load($default_layout_name);
+    // Skip any regions in the default layout that do not contain any blocks.
+    $default_layout_regions = array_filter($default_layout->positions);
+
     // Get the template being used by this layout.
     $template_info = layout_get_layout_template_info($layout->layout_template);
-    foreach ($default_layout->positions as $region => $region_block_list) {
-      if (array_key_exists($region, $template_info['regions'])) {
-        foreach ($region_block_list as $uuid) {
-          if (isset($default_layout->content[$uuid])) {
-            $block = $default_layout->content[$uuid];
+    $template_is_flexible = isset($template_info['flexible']) && $template_info['flexible'] == TRUE;
```

Comment:
`isset` && `true` is same as `!empty`

Reply:
Thanks @docwilmot ...fixed with [`08c297e` (#4704)](https://github.com/backdrop/backdrop/pull/4704/commits/08c297e09657de3c505c0fa0e4f85622f029b2ac)